tests/features/site.feature

```diff
@@ -11,7 +12,8 @@ Feature: site
 
   Scenario: Site Connection Mode
     @vcr site-connection-mode
-    Given a site named "[[test_site_name]]"
+    Given I am authenticated
+    And a site named "[[test_site_name]]"
     When I run "terminus site connection-mode --site=behat-test --env=dev"
```

Comment:
Oh, now I remember why I had to change the test and fixtures: the `site info` command works differently, and the the `Given a site named X` calls `sites info`, so I had to add additional info to the fixtures file. But I was _not_ trying to change the behavior or intent of the test.